stories/cosmoz-autocomplete.stories.js

```diff
@@ -107,16 +107,34 @@ export const wrap = () =>
 			style="max-width: 170px"
 		></cosmoz-autocomplete>`;
 
-export const select = () => html`
-	${css}
+export const select = () => html` ${css}
 	<cosmoz-autocomplete
 		.label=${'Choose color'}
 		.source=${colors}
 		.limit=${1}
 		.textProperty=${'text'}
-		.value=${colors[2]}
-		show-single
-		preserve-order
-		.min=${1}
-	></cosmoz-autocomplete>
-`;
+		.value=${[colors[0], colors[1], colors[2]]}
+		wrap
+		style="max-width: 170px"
+	></cosmoz-autocomplete>`;
+
+export const contour = () => html`<style>
```

Comment:
Add the font style css. It will work after upgrading cosmoz-input (https://neovici.slack.com/archives/C6LJQMJFM/p1711093197592089)

```suggestion
export const contour = () => html`${css}<style>
```

Reply:
Addressed, the font works as expected!